src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div className="Buttons">
+    <div className="group1">
+      <Button btname="AC" type="Button" />
+      <Button btname="+/-" type="Button" />
+      <Button btname="%" type="Button" />
+      <Button btname="/" type="Button" />
+    </div>
+    <div className="group2">
+      <Button btname="7" type="Button" />
+      <Button btname="8" type="Button" />
+      <Button btname="9" type="Button" />
+      <Button btname="X" type="Button" />
+    </div>
+    <div className="group3">
+      <Button btname="4" type="Button" />
+      <Button btname="5" type="Button" />
+      <Button btname="6" type="Button" />
+      <Button btname="-" type="Button" />
+    </div>
+    <div className="group4">
+      <Button btname="3" type="Button" />
+      <Button btname="2" type="Button" />
+      <Button btname="1" type="Button" />
+      <Button btname="+" type="Button" />
+    </div>
+    <div className="group5">
+      <Button btname="0" type="Button" />
+      <Button btname="," type="Button" />
+      <Button btname="." type="Button" />
+      <Button btname="=" type="Button" />
+    </div>
+  </div>
+);
+
+export default ButtonPanel;
```

Comment:
**[ OPTIONAL ]** For this part of your code, it would be better and more fluid if we render the buttons in a dynamic way, using map through an array containing the button names and rendering the btname prop through that.

Reply:
Will do that. thanks for your review. :+1: